### ISSUE TITLE
[Bug]: Sensor is mis-detected a IBeacon tracker

### ISSUE THREAD

**Issue by muellthosbutler** [custom-components/ble_monitor#1150]
### What happened?

After first activation (e.g. remove power insulation) the JHT was immediately detected as "Ibeacon Tracker". This repeats every several seconds:
![image](https://user-images.githubusercontent.com/85580534/229293723-c39a27e6-7671-4cb7-8ac5-3cf96b5597d4.png)
![image](https://user-images.githubusercontent.com/85580534/229293789-8f625cbc-749e-487f-a231-e4b116aade08.png)
I have then downloades the passive BLE Monitor custom component, installed and configured. But it doesn't detect the JHT, no matter if I use the GUI or making the manual entries in the config.yaml. Also, there are no logs for it.
My environment: HA OS on a dedicated Intel Pentium SBC, 2 BLE Proxies, several LYWSD03MMC using Xiaomi BLE integration.

### Sensor type

Jaalee Temperature and Humidity Sensor JHT

### Relevant log output

```shell
Sorry, no logs from passive BLE Monitor Integration
```

**Comment by Ernst79** [comment]
The iBeacon sensor is a sensor that is added via the Bluetooth and iBeacon integrations. 

Please not that running the Bluetooth integration and BLE monitor at the same time can cause troubles, often one of them doesn't work. 

Jalee JHT sensors requires that you use active scan. Did you enable active scan in the BLE monitor options?

**Comment by muellthosbutler** [comment]
Hello Ernst79I tried passive as well as active scan, both without success. You write, that using both passive BLE Monitor Integration and (Xiaomi BLE) Bluetooth Integration at the same time causes problems. I think that's the case here. Why is that? Any chance or plan to solve this?

**Comment by Ernst79** [comment]
This is caused by the fact that both BLE monitor and the Bluetooth integration want to use the BLE dongle. Especially is one integrations wants to use it in passive mode, while the other wants active mode, it gets confused, 

What we can do, is that you make a log in HA from the Bluetooth integration at debug level. Try to filter out one of the messages, and I can check what is wrong.

**Comment by muellthosbutler** [comment]
Here is the log snippet. I used :
  logs:
    homeassistant.components.bluetooth: debug
_________________________________________________________________-
2023-04-02 18:43:09.552 DEBUG (MainThread) [homeassistant.components.bluetooth.manager] BT-Proxy-2.OG (40:22:d8:ea:23:fc) [connectable]: A4:C1:38:50:5C:7E AdvertisementData(local_name='LYWSD03MMC', service_data={'0000fe95-0000-1000-8000-00805f9b34fb': b'XX[\x05\xa6~\\P8\xc1\xa4\xeb,\xbcc\x96\x1e\x01\x008\xce\x02\x18'}, tx_power=-127, rssi=-90) match: set()
2023-04-02 18:43:10.356 DEBUG (MainThread) [homeassistant.components.bluetooth.manager] BT-Proxy-2.OG (40:22:d8:ea:23:fc) [connectable]: D6:E9:62:EC:4F:FA AdvertisementData(manufacturer_data={76: b'\x02\x15\xeb\xef\xd0\x83p\xa2G\xc8\x987\xe7\xb5cM\xf5%`\x90m6\xcbd'}, service_data={'0000f525-0000-1000-8000-00805f9b34fb': b'd\xfaO\xecb\xe9\xd6`\x90m6'}, service_uuids=['0000f525-0000-1000-8000-00805f9b34fb'], tx_power=-127, rssi=-60) match: set()
2023-04-02 18:43:21.921 DEBUG (MainThread) [homeassistant.components.bluetooth.manager] BT-Proxy-2.OG (40:22:d8:ea:23:fc) [connectable]: A4:C1:38:50:5C:7E AdvertisementData(local_name='LYWSD03MMC', service_data={'0000fe95-0000-1000-8000-00805f9b34fb': b'0X[\x05\xa7~\\P8\xc1\xa4\x08'}, tx_power=-127, rssi=-87) match: set()
__________________________________________________

best regards, muellthosbutler :-)

**Comment by Ernst79** [comment]
Ok, I do see the Jaalee data. If I parse it manually, it reports. 

```
{'firmware': 'Jaalee', 'temperature': 19.43, 'humidity': 47.33, 'battery': 100, 'rssi': -52, 'mac': 'D6E962EC4FFA', 'type': 'JHT', 'packet': 'no packet id', 'data': True}
```

I think what is going on, let me try to explain. BLE messages can be received by the Bluetooth integration or by BLE monitor

**Bluetooth integration**
The Jaalee sensor is being received by the Bluetooth Proxy (BT-Proxy-2.OG). Bluetooth proxies forward the data to the Bluetooth integration on your HA machine, which will pass it on to the (BLE) integration that understands the data. The data contains two parts, a iBeacon message and a Jaalee message. There is no Jaalee BLE integration yet, only a iBeacon integration. That is why you will get an iBeacon sensor on your system, and no Jaalee sensor. 

**BLE monitor**
BLE monitor is able to receive Jaalee messages. However, BLE monitor has it's own Bluetooth engine, and wont work together with ESPHome Bluetooth proxies. That is probably why you also don't see it in BLE monitor. BLE monitor does work with ESPhome BLE Gateway, which has similar functionality. 

So, I'm afraid you will have to wait till we create a Jaalee integration in HA. In the meantime, try to move the sensor closer to your HA machine, to send the data directly to HA (BLE monitor), or use ESPhome BLE gateway. https://custom-components.github.io/ble_monitor/parse_data

**Comment by muellthosbutler** [comment]
So I have to wait for the Jaalee integration, because I want to monitor my
fridges with the JHT devices. There were only BLE Proxies in reach. Thanks.
I will make a feature request for that, maybe there are clothes who need
the integration also.
Best regards

Am So., 2. Apr. 2023 um 21:09 Uhr schrieb Ernst Klamer <
***@***.***>:

**Comment by Ernst79** [comment]
Yes, I'm afraid so. Or you could buy an extra ESP and install ESPhome BLE Gateway on that one.
